x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/read_index_route.ts

```diff
@@ -59,8 +59,8 @@ export const readIndexRoute = (
           );
 
           if (indexExists) {
-            let mappingOutdated: boolean | null = null;
-            let aliasesOutdated: boolean | null = null;
+            let mappingOutdated: boolean | undefined;
+            let aliasesOutdated: boolean | undefined;
```

Comment:
This looks like a breaking change, is it necessary?

Reply:
With nullable `index_mapping_outdated` it's not necessary so I rolled it back.